### ISSUE TITLE
TypeError: Converting circular structure to JSON

### ISSUE THREAD

**Issue by vjpr** [MauriceButler/bunyan-loggly#11]
I am logging a request from `koa-bunyan-logger`. When I use `bunyan-loggly` it errors like below. When I use a regular bunyan logger. No error.

```
TypeError: Converting circular structure to JSON
    at Object.stringify (native)
    at Bunyan2Loggly.write (xxx/node_modules/bunyan-loggly/index.js:24:32)
    at Logger._emit (xxx/node_modules/bunyan/lib/bunyan.js:939:22)
    at Logger.info (xxx/node_modules/bunyan/lib/bunyan.js:1044:24)
    at requestLogger (xxx/node_modules/koa-bunyan-logger/index.js:136:13)
    at dispatch (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:43:32)
    at next (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:44:18)
    at logger (xxx/node_modules/koa-bunyan-logger/index.js:39:12)
    at dispatch (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:43:32)
    at next (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:44:18)
    at logger (xxx/node_modules/koa-logger/index.js:49:12)
    at dispatch (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:43:32)
    at next (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:44:18)
    at xxx/node_modules/koa-mount-hot/index.js:116:26
    at dispatch (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:43:32)
    at next (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:44:18)
    at xxx/node_modules/koa-mount-hot/index.js:116:26
    at dispatch (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:43:32)
    at next (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:44:18)
    at xxx/node_modules/koa-mount-hot/index.js:116:26
    at dispatch (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:43:32)
    at next (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:44:18)
    at xxx/node_modules/koa-mount-hot/index.js:116:26
    at dispatch (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:43:32)
    at next (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:44:18)
    at xxx/node_modules/koa-mount-hot/index.js:116:26
    at dispatch (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:43:32)
    at next (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:44:18)
    at xxx/node_modules/koa-mount-hot/index.js:116:26
    at dispatch (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:43:32)
    at next (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:44:18)
    at xxx/node_modules/koa-mount-hot/index.js:116:26
    at dispatch (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:43:32)
    at next (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:44:18)
    at xxx/node_modules/koa-mount-hot/index.js:116:26
    at dispatch (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:43:32)
    at next (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:44:18)
    at xxx/node_modules/koa-mount-hot/index.js:116:26
    at dispatch (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:43:32)
    at next (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:44:18)
    at xxx/node_modules/koa-mount-hot/index.js:116:26
    at dispatch (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:43:32)
    at next (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:44:18)
    at xxx/node_modules/koa-mount-hot/index.js:116:26
    at dispatch (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:43:32)
    at next (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:44:18)
    at xxx/node_modules/koa-mount-hot/index.js:116:26
    at dispatch (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:43:32)
    at next (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:44:18)
    at xxx/node_modules/koa-mount-hot/index.js:116:26
    at dispatch (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:43:32)
    at next (xxx/node_modules/live-koa-server/node_modules/koa/node_modules/koa-compose/index.js:44:18)
```

**Comment by MauriceButler** [comment]
Hi @vjpr 

This was due to a naive stringify method.

I have now switched to use the same method as the underlying loggly module.

**Comment by MauriceButler** [comment]
Released in [v1.2.0](https://github.com/MauriceButler/bunyan-loggly/releases/tag/v1.2.0)
